packages/compile-solidity/package.json

```diff
@@ -1,4 +1,4 @@
-{
+  {
```

Comment:
Really minor note, but, you seem to have accidentally introduced an extra space here?

Reply:
Thanks for catching this @haltman-at. I have fixed this in my new commit.